### ISSUE TITLE
Filter Verbosity

### ISSUE THREAD

**Issue by SpilledMilkFactory** [Game-Loom/GameLoom#33]
Verbosity Tweaks and Testing for Filtering functionalities.

From Testing Feedback Deliverable:

"Currently, the filtering in some fields appears to be case sensitive and it would be less burdensome for the users if they could enter the desired attribute without having to recall the exact capitalization."

- Possible user error, additional testing and tweaking as desired based on findings

***UPDATES:***
- User interpretation likely resulted in thinking it was a case sensitive issue (ignoreCase is used everywhere in the section)

**What we now know:**

- In the main library windows when setting the sort and filter then clicking sort and filter with known values the box will go blank -> it is not until you click the "search" button again that you will see your results, and those results are consistently correct.

- Basically clicking the sort and filter button sets the filters and sorting options, but clears whats on screen and starts the user with a blank slate that has the filters applied for them to search within again

- Not sure if that's actually a bug or even something we can fix, but we can add something about it in known limitations if its a JavaFX hang up
